### ISSUE TITLE
Minimize height of upload changes dialog

### ISSUE THREAD

**Issue by johnjohndoe** [MarcusWolschon/osmeditor4android#790]
I noticed that the changes dialog does not show a lot of changes most of the time (at least for me). The same is true "Properties" tab.

![changes](https://user-images.githubusercontent.com/144518/46560299-72986c00-c8f3-11e8-800b-d1e391e2bf00.png)

# Proposal

Therefore, I though it might make sense to define a minimum size or even set `android:layout_height="wrap_content"` to let the dialog shrink to the space needed by its content.

# Benefits

The partial benefit would be that the keyboard does not block the upload buttons in some cases. This of course depends on the device size.

# Related

- Pull request #579

**Comment by simonpoole** [comment]
Haven't got around to thinking about this yet.

**Comment by johnjohndoe** [comment]
No hurries! Just a _written_ reminder ;-)

**Comment by simonpoole** [comment]
I'll probably have a further look at this when I add some more things here, in particular some kind of social media and hashtag support.

**Comment by simonpoole** [comment]
Both iD and JOSM have facilities for adding hashtags in an additional changeset tag (it doesn't really matter if it is a good idea or not, at least it helps trying to keep the normal comment field meaningful).

The other aspect is in some way integrating information from https://github.com/osmlab/osm-community-index

**Comment by johnjohndoe** [comment]
Thank you for the clarification. I did not know about the _osm-community-index_ before.
Which would be the additional changeset tag? Is there a wiki proposal?

**Comment by simonpoole** [comment]
IMHO this is now moot with the larger EditTexts and additional content

**Comment by johnjohndoe** [comment]
I also think the screen is fine now. Here is a screenshot to document the current implementation (13.1.6).

![Vespucci1](https://user-images.githubusercontent.com/144518/64064315-08ba3c80-cc00-11e9-9ad6-60cfa77603c9.png)

I particular like the `x` to delete texts. Thank you for this feature.

One major usability issue which I noticed for me was on how to get to the upload button. I realized that tapping the blue button in the bottom right corner twice actually minimizes the keyboard. Then it is easy to tap the upload button. :flushed: 

![Vespucci2](https://user-images.githubusercontent.com/144518/64064317-0e178700-cc00-11e9-9ecb-e3f2ab566102.png)
